packages/camera/camera_avfoundation/ios/camera_avfoundation/Sources/camera_avfoundation/FLTCaptureConnection.m

```diff
@@ -4,11 +4,9 @@
 
 #import "./include/camera_avfoundation/FLTCaptureConnection.h"
 
-@interface FLTDefaultCaptureConnection ()
-@property(nonatomic, strong) AVCaptureConnection *connection;
-@end
-
-@implementation FLTDefaultCaptureConnection
+@implementation FLTDefaultCaptureConnection {
+  AVCaptureConnection *_connection;
```

Comment:
i think we prefer properties over ivars. Can you keep the property, and use dot syntax at call site instead?

Reply:
I updated the other wrapper as well